packages/client-js/src/Web3API.ts

```diff
@@ -102,14 +104,20 @@ export class Web3API {
       const { name, depth, type, path } = file;
 
       if (depth === 1 && type === "file" &&
-         (name === "web3api.yaml" || name === "web3apy.yml")) {
+         (name === "web3api.yaml" || name === "web3api.yml")) {
         const manifestStr = await portals.ipfs.catToString(path);
         this._manifest = YAML.safeLoad(manifestStr) as Manifest | undefined;
 
         if (this._manifest === undefined) {
           throw Error(`Unable to parse web3api.yaml\n${manifestStr}`);
         }
 
+        const latestVersion: string = require("../package.json").version
```

Comment:
Instead of doing run-time requiring of the package.json, let's try to do this instead inside of the `manifest/versions` directory let's add a `latest.ts` file that contains:
```typescript
export const version: string = "0.0.1-alpha.1";
```